sweep/sweeper.go

```diff
@@ -221,6 +221,30 @@ func (p *SweeperInput) terminated() bool {
 	}
 }
 
+// isMature returns a boolean indicating whether the input has a timelock that
+// has been reached or not. The locktime found is also returned.
+func (p *SweeperInput) isMature(currentHeight uint32) (bool, uint32) {
```

Comment:
How do timelocked inputs get to the sweeper?  I thought they were only supposed to be offered to the sweeper once timelocks are fulfilled.